Dockerfile.blackhole-ui

```diff
@@ -0,0 +1,46 @@
+# Frontend build stage
+FROM node:18-slim AS frontend-builder
+WORKDIR /app
+
+# Copy package files first for better caching
+COPY client/package*.json ./
+RUN npm install
+
+# Copy the rest of the client code
+COPY client/ ./
+
+# Build the frontend first
+RUN npm run build
+
+# Create directory structure for mediainfo images
+RUN mkdir -p /app/build/static/images/mediainfo/codec \
+    /app/build/static/images/mediainfo/edition \
+    /app/build/static/images/mediainfo/resolution
+
+# Copy mediainfo images to the correct location and verify
+COPY client/src/images/mediainfo/codec/* /app/build/static/images/mediainfo/codec/
+COPY client/src/images/mediainfo/edition/* /app/build/static/images/mediainfo/edition/
+COPY client/src/images/mediainfo/resolution/* /app/build/static/images/mediainfo/resolution/
```

Comment:
This can probably be simplified

Reply:
Or removed? It doesn't seem like we're using the static folder here.

Reply:
It’s for the react build of the front end